node/mo/package.json

```diff
@@ -3,16 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 80",
```

Comment:
Why change this back to 3000?

Reply:
Oh, that was just for me locally. I didn't mean to add that change.